cloudtools/scripts/aws_create_ami.py

```diff
@@ -211,15 +211,11 @@ def create_ami(host_instance, args, config, instance_config, ssh_key,
     else:
         with lcd(config_dir):
             put('etc/yum-local.cfg', '%s/etc/yum-local.cfg' % mount_point)
-            put('groupinstall', '/tmp/groupinstall')
-            put('additional_packages', '/tmp/additional_packages')
         yum = 'yum -d 1 -c {0}/etc/yum-local.cfg -y --installroot={0} '.format(
             mount_point)
-        # groupinstall emulates the %packages section of the kickstart config
-        run('%s groupinstall "`cat /tmp/groupinstall`"' % yum)
-        # and this attempts to emulate the additional packages that Anaconda installs
-        # as it needs them.
-        run('%s install `cat /tmp/additional_packages`' % yum)
```

Comment:
True, this makes `aws_create_ami` unable to create CentOS-6.2 AMIs anymore -- but there's only one CentOS 6.5 AMI.

Yes, kernel, grub, etc. are included in Core - I verified that all of the packages that used to be in additional_packages get installed.